lua/astrocommunity/test/nvim-coverage/nvim-coverage.lua

```diff
@@ -0,0 +1,8 @@
+return {
+  "andythigpen/nvim-coverage",
+  lazy = false,
+  requires = { "nvim-lua/plenary.nvim" },
+  -- Optional: needed for PHP when using the cobertura parser
+  rocks = { "lua-xmlreader" },
+  config = function() require("coverage").setup() end,
```

Comment:
```suggestion
```

Lazy automaticially calls the setup function even if `config` is not defined. [See here](https://github.com/folke/lazy.nvim#-plugin-spec)